static/src/javascripts/projects/common/modules/spacefinder-debug-tools.js

```diff
@@ -19,6 +19,10 @@ const markCandidates = (exclusions, winners, options) => {
 			colour: colours.orange,
 			reason: 'Too close to top or bottom of article',
 		},
+        startAt: {
+            colour: colours.orange,
+            reason: 'Skipped because start at was specified',
```

Comment:
I 'm not fully understanding what this means. Could this be a little more descriptive like the other reasons pls?

Reply:
Hopefully it makes sense now!

Reply:
It does - thanks!